core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -428,6 +428,10 @@ angular.module('oppia').directive('explorationEditorPage', [
                 explorationData.email_preferences
                   .mute_suggestion_notifications);
 
+              var dependencyHTML = $.parseHTML(
```

Comment:
Can we use some method from angular instead here?

Reply:
Also, look into underscore library.

Reply:
I tried checking this, all the and it seems jQuery will be best for this.
underscore and angular don't have a method for this.